servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/NoopDnsCnameCache.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.dns.discovery.netty;
+
+import io.netty.channel.EventLoop;
+import io.netty.resolver.dns.DnsCnameCache;
+
+import javax.annotation.Nullable;
+
+/**
+ * {@link DnsCnameCache} implementation that does not cache CNAME entries.
+ */
+final class NoopDnsCnameCache implements DnsCnameCache {
```

Comment:
Can we use this implementation provided by netty itself?

Reply:
@NiteshKant we can... I just didn't do it because the one of netty will be `@Deprecated` in the next release... Let me know if you prefer to still use it tho.

Reply:
We can use the netty one and then remove this whole explicit setting of noop cache once netty is released.

Reply:
done